### DISCUSSION TITLE
localurl.test issues

### DISCUSSION THREAD

**Discussion by Ochab** [statamic/cms#5111] [Q&A] [answered]
I hate asking this question. It's probably been asked before but I can't find a solution since V3.

I'm on a new MacBook Pro my work sent me. Brand new user, no dev environments set up. Followed the setup tutorial at https://statamic.dev/installing/local. 

What I did:

1. Homebrew
2. php via homebrew
3. composer via homebrew.
4. `composer global require statamic/cli` (Had to troubleshoot zsh issues here, nbd)
5. `statamic new my-project`. (this was done in myusername/Sites)
6. Add superuser.

`http://my-project.test` does not work. When statamic used `laravel` this could be corrected using `valet park`. Is there a similar command I can use to set myusername/Sites as the root?

**Comment by jasonvarga** [comment]
If you park your sites directory it should notice your statamic project.

Did you park it already? If not, run `valet park` in there.

**Reply by Ochab** [reply]
Thanks Jason! Tried that:

`zsh: command not found: valet`

Is Valet something I should have installed? It wasn't listed in the Requirements so I have not set it up, and since it wasn't installed via the package manager I wasn't sure if that was the right solution. (I mentioned this in the original message)

**Comment by Ochab** [comment, marked as answer]
For anyone else who runs into this, I ended up installing valet. https://laravel.com/docs/8.x/valet#installation

I already had composer installed, so:

`composer global require laravel/valet`

`valet install`

Navigate toward your main web dev server (in my case `/Sites`) and use: `valet park`

Worked fine after that! A bit confused why I had to do this step though since it wasn't included in the prerequisites in the statamic install instructions.

**Reply by jasonvarga** [reply]
Valet isn't a prerequisite, you can use whatever server you want. Glad you figured it out though!
